spec/facets/DatasetVersionDatasetFacet.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/DatasetVersionDatasetFacet.json",
+  "$defs" : {
+    "DatasetVersionDatasetFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/DatasetFacet"
+        },
```

Comment:
Wasn't the original version of this facet capable of being treated as an input facet or an output facet? Do we not need that capability anymore?

Reply:
There were no changes in the facet since _original_ version I got here:
https://github.com/OpenLineage/OpenLineage/pull/485/files#diff-93680b419816322d4d4b9576e763f72d5ffc9c2c4d72569a8c7a64e1fa0c6843R8